### ISSUE TITLE
Low Bitrate? High Compression?

### ISSUE THREAD

**Issue by Mafiu0** [FlavioFS/ParsecSoda#26]
Hello,

So I have tried connecting to my own room created with Parsec Soda and I noticed that the video quality looks very compressed/low bitrate. I have tried increasing the bandwidth limit in the application but it doesn't seem to make much of a difference. 

I do not think it is necessarily a bandwidth issue on my end because if I connect directly to my computer (through the parsec app) the video will be in very high quality and with solid ms.

Is there a setting I may be missing within the program?

**Comment by LoneWolf1987** [comment]
Same about no matter what bandwidth limit you set, When guests more than 2, they get around 2 mbps each, but must be 10 per guest (if i set bandwidth to 30 mbps for 3 guests).

**Comment by Mafiu0** [comment]
Yeah, it's almost as it defaults to the **minimum bitrate** regardless of the bandwidth limit you set. If I set it to 50mbs my friend should get 25mbs more or less, but they get around 2-3mbs.

**Comment by CollinCodez** [comment]
The bandwidth _Limit_ is just that: a limit. It is not a target bandwidth, it is a limit. Parsec will only use as much bandwidth as it actually needs.

Parsec also has automatic congestion control that will limit the bandwidth more when it encounters network issues and congestion, and the parsec SDK (which is what Parsec Soda uses to host on parsec) has not been updated to use the newer congestion algorithms that help using more bandwidth.

There is nothing that can be done on the end of Parsec Soda, it is a matter of ensuring you don't have network issues and waiting for the Parsec Team to update the Parsec SDK.
